modules/local/input_tol.nf

```diff
@@ -0,0 +1,31 @@
+process INPUT_TOL {
+    label 'process_single'
+
+    conda (params.enable_conda ? "conda-forge::gawk=5.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+        'ubuntu:20.04' }"
```

Comment:
No we want it to be ubuntu, that was recommended to me. We use gawk/sed for conda, because there is no equivalent ubuntu package.

Example: https://github.com/nf-core/modules/blob/master/modules/nf-core/gunzip/main.nf